ui/src/assets/i18n/en.json

```diff
@@ -728,6 +728,7 @@
   "workflow_vcs_resynced": "VCS status resynced",
   "workflow_added": "Workflow added",
   "workflow_run_scheduling": "Crafting workflow",
+  "workflow_run_conditions_hook": "Pay attention, here you can't use run conditions on workflow build parameters because the conditions checks are executed before the workflow run is created",
```

Comment:
```suggestion
  "workflow_run_conditions_hook": "Pay attention, you can not use run conditions with variable as {{.cds.build...}} because conditions are checked before a run is created.",
```